### PR TITLE
render service test data with different function forms of `getRuntimeWithModelQueryConnection`

### PR DESCRIPTION
**Pull request by YannanGao-gs**
## Summary

render service test data with different function forms of `getRuntimeWithModelQueryConnection`

## How did you test this change?

- [x] Test(s) added
- [x] Manual testing (please provide screenshots/recordings)
- [ ] No testing (please provide an explanation)

before: 

https://github.com/finos/legend-studio/assets/73408381/d8ab9d93-8ac1-4398-8f0d-97031c7e5cc5

after: 

https://github.com/finos/legend-studio/assets/73408381/cdb60b2f-dbc1-4999-b58c-1965e181bb43

### CODE REVIEW

File: packages/legend-application-studio/src/components/editor/editor-group/service-editor/testable/ServiceTestsEditor.tsx [hunk 1]
```diff
@@ -147,8 +147,8 @@ export const ExternalFormatParameterEditorModal = observer(
     isReadOnly: boolean;
     onClose: () => void;
     updateParamValue: (val: string) => void;
-    bindingParamPair: {
-      binding: Binding;
+    bindingOrContentTypeParamPair: {
+      candidate: Binding | string;
```
Comment (MauricioUyaguari) [inline packages/legend-application-studio/src/components/editor/editor-group/service-editor/testable/ServiceTestsEditor.tsx:151]: maybe im wrong but isnt whats important here just the content type so we can render the text in the correct format. 
Why do we need the binding. could we just not resolve the content type of the binding here and just keep it contentType, param for the ParamPairs?

File: packages/legend-application-studio/src/components/editor/editor-group/service-editor/__tests__/ServiceEditorTest.test.tsx [hunk 2]
```diff
@@ -144,17 +145,22 @@ test(
...
+      .testableState.selectedSuiteState?.testStates[0]?.setupState.getBindingOrContentTypeWithParamFromQuery();
     expect(bindingParmPairs).toHaveLength(2);
     const firstPair = guaranteeNonNullable(
       guaranteeNonNullable(bindingParmPairs)[0],
     );
     const secondPair = guaranteeNonNullable(
       guaranteeNonNullable(bindingParmPairs)[1],
     );
-    expect(firstPair.binding.name).toBe('PersonBinding');
+
```
Comment (MauricioUyaguari) [inline packages/legend-application-studio/src/components/editor/editor-group/service-editor/__tests__/ServiceEditorTest.test.tsx:156]: please add a test specific to the PR here

File: packages/legend-application-studio/src/components/editor/editor-group/service-editor/testable/ServiceTestsEditor.tsx (hunk 1 above)
Comment (YannanGao-gs) [reply packages/legend-application-studio/src/components/editor/editor-group/service-editor/testable/ServiceTestsEditor.tsx]: a heads up: contentType from Binding is somehow undefined in the test (only in the test)

File: packages/legend-application-studio/src/components/editor/editor-group/service-editor/__tests__/ServiceEditorTest.test.tsx (hunk 2 above)
Comment (YannanGao-gs) [reply packages/legend-application-studio/src/components/editor/editor-group/service-editor/__tests__/ServiceEditorTest.test.tsx]: test is added

File: packages/legend-application-studio/src/components/editor/editor-group/service-editor/testable/ServiceTestsEditor.tsx (hunk 1 above)
Comment (YannanGao-gs) [reply packages/legend-application-studio/src/components/editor/editor-group/service-editor/testable/ServiceTestsEditor.tsx]: fixed it by loading a plugin

**Pull request merged**